homeassistant/components/glances/__init__.py

```diff
@@ -173,4 +173,7 @@ def get_api(hass, entry):
     """Return the api from glances_api."""
     params = entry.copy()
     params.pop(CONF_NAME)
+
+    # popping CONF_VERIFY_SSL for backward compatibility after it was deprecated
+    params.pop(CONF_VERIFY_SSL, false)
```

Comment:
```suggestion
    params.pop(CONF_VERIFY_SSL, None)
```

Reply:
Thanks

Reply:
This what happens when you give a Java developer to write in Python 😆